packages/cicero-cli/test/cli.js

```diff
@@ -940,9 +940,33 @@ describe('#validateArchiveArgs', () => {
 });
 
 describe('#archive', async () => {
+    it('should create signed archive', async () => {
+        const archiveName = 'test.cta';
+        const p12path = path.resolve(__dirname, 'data/keystore.p12');
+        const keystore = {
+            path: p12path,
+            passphrase: 'password'
+        };
+        const options = {
+            keystore: keystore
+        };
+        const result = await Commands.archive(template, 'ergo', archiveName, options);
+        result.should.eql(true);
+        const newTemplate = await Template.fromArchive(fs.readFileSync(archiveName));
+        newTemplate.should.not.be.null;
+        newTemplate.should.have.own.property('authorSignature');
+        fs.unlinkSync(archiveName);
+    });
+
     it('should create a valid ergo archive', async () => {
         const archiveName = 'test.cta';
-        const result = await Commands.archive(template, 'ergo', archiveName);
+        const options = {
```

Comment:
We shouldn't need to pass any options if there is no keystone. This should work with `options = null` or `options = {}`. This would also mean that all existing tests should pass without changes.

This comment applies similarly to many of the other changes to CLI tests.